exla/Makefile

```diff
@@ -1,15 +1,15 @@
 # Environment variables passed via elixir_make
 # ERTS_INCLUDE_DIR
+# MIX_APP_PATH
 
 # XLA Extension Installation Location
 XLA_EXTENSION_DIR ?= cache/xla_extension
-
 XLA_EXTENSION_LIB = $(XLA_EXTENSION_DIR)/lib
 XLA_INCLUDE_PATH = $(XLA_EXTENSION_DIR)/include
 
 # Private configuration
 EXLA_DIR = c_src/exla
-PRIV_DIR = priv
+PRIV_DIR = $(MIX_APP_PATH)/priv
```

Comment:
By using environment specific priv we need to compile EXLA separately for dev and test, while currently we compile into top-level priv that gets linked in all environments. Is this the desired behavoiur?

Reply:
@jonatanklosko yes, it was on purpose because nuking _build should recompile this stuff and we should avoid doing things in priv.